From: Daniel Leidert <dleidert@debian.org>
Description: Fix a syntax warning
 Fix "SyntaxWarning: assertion is always true, perhaps remove parentheses?"
Bug-Debian: https://bugs.debian.org/641788

--- a/rdkit/sping/PDF/pdfutils.py
+++ b/rdkit/sping/PDF/pdfutils.py
@@ -29,7 +29,7 @@
     code.append('ID')
     #use a flate filter and Ascii Base 85
     raw = img.tostring()
-    assert(len(raw) == imgwidth * imgheight, "Wrong amount of data for image")
+    assert len(raw) == imgwidth * imgheight, "Wrong amount of data for image"
     compressed = zlib.compress(raw)   #this bit is very fast...
     encoded = _AsciiBase85Encode(compressed) #...sadly this isn't
     
